lstmcpipe/stages/mc_dl2_to_sensitivity.py

```diff
@@ -126,7 +126,7 @@ def dl2_to_sensitivity(
 
     # Create plot from sensitivity files
     base_cmd_plot = (
-        f'lstmcpipe_plot_irfs -f {Path(output).parent} -o {output.replace(".fits.gz", ".png")}'
```

Comment:
> Currently we are creating the sensitivity plots in the same stage we are running the `lstmcpipe_dl2_to_sensitivity`. I think we can adapt PathConfig to create the plots in the same stage rather than adding an extra stage. What do you think ?

Sounds good.